composer.json

```diff
@@ -1,44 +1,48 @@
 {
-	"name":"contao-community-alliance/events-cron",
-	"description":"Cron event for Contao Open Source CMS",
-	"keywords":["contao", "event", "cron"],
-	"type": "contao-module",
-	"license":"LGPL-3.0+",
-	"authors":[
-		{
-			"name":"Tristan Lins",
-			"email":"tristan.lins@bit3.de",
-			"homepage":"http://bit3.de",
-			"role":"Developer"
-		}
-	],
-	"support":{
-		"email":"info@contao-community-alliance.de",
-		"issues":"https://github.com/contao-community-alliance/events-cron/issues",
-		"wiki":"https://github.com/contao-community-alliance/events-cron/wiki",
-		"irc":"irc://irc.freenode.org/contao.dev",
-		"source":"https://github.com/contao-community-alliance/events-cron"
-	},
-	"require":{
-		"php":">=5.3",
-		"contao/core": ">=2.11,<4.0-dev",
-		"contao-community-alliance/composer-plugin": "~2.0",
-		"contao-community-alliance/event-dispatcher": "~1.0"
-	},
-	"autoload":{
-		"psr-0":{
-			"ContaoCommunityAlliance\\Contao\\Events\\Cron": "src/"
-		}
-	},
-	"extra": {
-		"contao": {
-			"sources": {
-				"contao": "system/modules/events-cron"
-			}
-		},
-		"branch-alias": {
-			"dev-master": "1.0.x-dev",
-			"dev-develop": "1.1.x-dev"
-		}
-	}
+  "name": "contao-community-alliance/events-cron",
+  "description": "Cron event for Contao Open Source CMS",
+  "keywords": [
+    "contao",
+    "event",
+    "cron"
+  ],
+  "type": "contao-bundle",
+  "license": "LGPL-3.0-or-later",
+  "authors": [
+    {
+      "name": "Tristan Lins",
+      "email": "tristan.lins@bit3.de",
+      "homepage": "http://bit3.de",
+      "role": "Developer"
+    }
+  ],
+  "support": {
+    "email": "info@contao-community-alliance.de",
+    "issues": "https://github.com/contao-community-alliance/events-cron/issues",
+    "wiki": "https://github.com/contao-community-alliance/events-cron/wiki",
+    "irc": "irc://irc.freenode.org/contao.dev",
+    "source": "https://github.com/contao-community-alliance/events-cron"
+  },
+  "require": {
+    "php": "^5.6 || ^7.0",
+    "contao/core-bundle": "~4.4",
+    "symfony/dependency-injection": "^3.4 || ^4.0",
+    "symfony/event-dispatcher": "^3.4 || ^4.0",
+    "symfony/http-kernel": "^3.4 || ^4.0"
+  },
+  "require-dev": {
+    "contao/manager-plugin": "^2.2"
+  },
+  "autoload": {
+    "psr-4": {
+      "ContaoCommunityAlliance\\Contao\\Events\\Cron\\": "src/"
+    }
+  },
+  "extra": {
+    "contao-manager-plugin": "ContaoCommunityAlliance\\Contao\\Events\\Cron\\ContaoManager\\Plugin",
+    "branch-alias": {
+      "dev-master": "1.0.x-dev",
+      "dev-develop": "1.1.x-dev"
```

Comment:
Bump branch alias

Reply:
What versions to use?

Reply:
1.1 for Contao 4, no?

Reply:
When this package has no bc breaks the new version is (master 1.1 develop 1.2).
If use has bc breaks then is (master 2.0 develop 2.1)

Reply:
I think better is 2.0. The actual version is installable for contao 4.4